Providers/FacebookUserProfileInfoProvider.cs

```diff
@@ -7,17 +7,26 @@
 
 namespace PMAuth.Providers
 {
+    /// <summary>
+    /// Provider from FacebookInfoModel to UserProfile
+    /// </summary>
     public static class FacebookUserProfileInfoProvider
     {
+        /// <summary>
+        /// Mapps UserProfile fields
+        /// </summary>
+        /// <param name="facebookInfo">FacebookInfoModel</param>
+        /// <returns>UserProfile</returns>
         public static UserProfile Provider(FacebookInfoModel facebookInfo)
```

Comment:
Are you sure that facebookInfo always not null?

Reply:
Not sure, need to add validation